clippy_lints/src/transmuting_null.rs

```diff
@@ -0,0 +1,114 @@
+use crate::consts::{constant_context, Constant};
+use crate::utils::{match_qpath, span_lint};
+use if_chain::if_chain;
+use rustc::hir::{Expr, ExprKind};
+use rustc::lint::{in_external_macro, LateContext, LateLintPass, LintArray, LintContext, LintPass};
+use rustc::{declare_tool_lint, lint_array};
+use syntax::ast::LitKind;
+
+declare_clippy_lint! {
+    /// **What it does:** Checks for transmute calls which would recieve a null pointer.
```

Comment:
```suggestion
    /// **What it does:** Checks for transmute calls which would receive a null pointer.
```

Reply:
Ah, I knew it, I always get them the other way around! >w< (thank you!)

Reply:
Corrected!